.gitlab-ci.yml

```diff
@@ -225,6 +225,13 @@ build:egde:dune:dev:
     OPAM_SWITCH: edge
     DUNE_TARGET: world
 
+build:asyncproofs:
```

Comment:
Maybe you can just reuse the edge build for this?

Anyways, the naming is not good, it should be `build:base+async`